themes/default/content/docs/intro/languages/yaml.md

```diff
@@ -42,7 +42,7 @@ The fastest way to get up and running is to choose from one of the following Get
             <img class="h-8 mx-auto" src="/logos/tech/gcp.svg" alt="Google Cloud">
         </a>
     </div>
-    <div class="flex-1 pb-4">
+    <div class="flex-1 pb-4 md:mr-4">
```

Comment:
Since this is the last item in the horizontal list, we leave this extra spacing out, so the rightmost edges line up properly -- otherwise you end up with a little gap:
 
![image](https://user-images.githubusercontent.com/274700/166972766-5898c340-fc90-49d0-bbc0-273f46c84a22.png)


```suggestion
    <div class="flex-1 pb-4>
```